Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -1081,7 +1081,7 @@ void VideoConfigDiag::PopulatePostProcessingShaders()
   if (shaders.empty())
     return;
 
-  choice_ppshader->AppendString(_("(off)"));
+  choice_ppshader->AppendString(_("Off"));
```

Comment:
I think it's a bit better if we change it back, but I'm fine with having it the new way if someone else thinks that's ideal.